articles/policies/entity-limits.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Entity Limit Policy
+description: Describes Auth0's tenant entity limit policy for subscribers.
+topics:
+    - auth0-policies
+    - rate-limits
+    - entity-limits
+contentType:
+  - reference
+useCase:
+  - support
+---
+# Entity Limit Policy
+
+Entities in Auth0 are tenant configuration elements such as applications, connections, rules, and API resource servers. 
+
+Auth0 limits the number of entities you can have depending on your subscription level. Auth0 provides notifications to you when you are approaching (90%) and when you have reached (100% or higher) your respective entity limits. We will also provide informative messages to prevent you from attempting to configure entities that would end up being rejected.
```

Comment:
```suggestion
Auth0 limits the number of entities you can have depending on your subscription level. Auth0 provides notifications to you when you are approaching (80%) and when you have reached (100% or higher) your respective entity limits. We will also provide informative messages to prevent you from attempting to configure entities that would end up being rejected.
```

Reply:
manually updated with a separate commit